### DISCUSSION TITLE
GPT-SoVITS Support

### DISCUSSION THREAD

**Discussion by RoversX** [DrewThomasson/ebook2audiobook#145] [Ideas]
Great project! Thank you! Will other TTS models like GPT-SoVits be supported in the future?
Cheers!
https://github.com/RVC-Boss/GPT-SoVITS

**Comment by ROBERT-MCDOWELL** [comment]
why not.... on the tts list. thanks

**Comment by ROBERT-MCDOWELL** [comment]
well, after review this repo it looks like ebook2audiobook project, they use what we use already....
